pkgs/applications/audio/ams/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchgit
+, automake
+, alsaLib
+, ladspaH
+, libjack2
+, fftw
+, zita-alsa-pcmi
+, qt5
+, pkg-config
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "ams";
+  version = "unstable-2019-04-27";
+
+  src = fetchgit {
+    url = https://git.code.sf.net/p/alsamodular/ams.git;
+    sha256 = "0qdyz5llpa94f3qx1xi1mz97vl5jyrj1mqff28p5g9i5rxbbk8z9";
```

Comment:
Without `rev` this would always fetch `HEAD` which can change.

Reply:
Fetching and fixing this...